backend/src/main/java/com/crm/exception/UserDisabledException.java

```diff
@@ -0,0 +1,8 @@
+package com.crm.exception;
+
+public class UserDisabledException extends RuntimeException {
```

Comment:
Dobrze by było zacząć cały kod przepisywać na kod domenowy. 
Wyglądałoby to tak że w crm mamy paczke np. User w nim paczki takie jak controller, service, repository, exception itp. 
Z biegiem czasu ilość klas nas dobije i zaczniemy się gubić. Warto pomyśleć o stworzeniu jakiegoś oddzielnego zadanka na to.

Reply:
To co piszesz ma związek z **Domain Driven Design**?
Dodam teraz do backloga takie zadanie. Pózniej się je dokądniej opisze.
Wspomnę o tym na spotkaniu w pon.